lms/djangoapps/course_api/helpers.py

```diff
@@ -61,4 +65,5 @@ def get_marketing_data(course_key, language):
         log.warning('Could not fetch the marketing details from the API. course_key=[%s], status_code=[%s], url=[%s].',
                     course_key, response.status_code, url)
         return {}
+    cache.set(CACHE_KEY, response.json(), 30 * 60)
```

Comment:
Please inform QA that this change will cause the marketing changes to only be reflected on eps that consume this after 30 minutes. Maybe thats a long time no? Or you could clear cache when a marketing change happens (not the best solution to be honest, i hope in the future we avoid this whole pattern)

Reply:
@Dabbas on it